Mike/formCreate.js

```diff
@@ -0,0 +1,30 @@
+// formCreate.js
+// Creator Michael Sewell
+
+exports.convert = async (obj) => {
+    let formString = "<form>"
+
+    if (obj) { // error handling
+        await obj.forEach(element => {
+            
+            const type = element.type
+            if (type == "label") {
+                formString += `<label for="${element.for}"></label>`;
+            } else if (type == "text" || type == "color" || type == "email" || type == "tel") {
```

Comment:
looks like you forgot to add the input `type` to `<input id="${element.name}" name="${element.name}" />`

Reply:
You are right, I must have overlooked that. Thanks for the catch!

Reply:
overall everything looks good, that update should do it.

Reply:
I fixed the error, let me know if that looks good! Thanks in advance.

Reply:
looks like you already assigned a variable `const type` so you can just use `${type}` instead of `type="${element.type}"`

Reply:
since type already equals element.type